### ISSUE TITLE
Is it possible to run a single benchmark rather than all the benchmark tests?

### ISSUE THREAD

**Issue by bahjat1993** [lpereira/hardinfo#691]
As the title states, is it possible to run a single bench mark test rather than all the tests? 
Also i've noticed that in the readme file and the help it doesn't state that you -rma is a valid arg but when i send the command _hardinfo -rma devices.so_ it seems to be fine with it.

**Comment by lpereira** [comment]
Yes, pass the `-b` option. For instance, `hardinfo -b "CPU Fibonacci"`
